examples/non-transaction-retry/non_transaction_retry.bal

```diff
@@ -21,8 +34,26 @@ int count = 0;
 function doWork() returns error? {
     if count < 1 {
         count += 1;
-        return error("Execution Error");
+        // Return a retriable error so that
+        // the default retry manager retries.
+        return errors:Retriable("Execution Error");
     } else {
-        io:println("Work done.");
+        io:println("Work is done.");
     }
 }
+
+// Retry manager class with arbitrarily logic.
```

Comment:
```suggestion
// Retry the manager class with an arbitrarily logic.
```

Reply:
this is to say, pass a retry manager class which can have any arbitrary logic